index.php

```diff
@@ -95,7 +95,7 @@ private function build_index() {
 
 			$uri = trim( sprintf( 
 					'%s/%s',
-					trim( $uri_info['dirname'], '/' ),
+					trim( $uri_info['dirname'], '/\\' ),
```

Comment:
One `\` means to escape next character, which is `'`. Giving two, first `\` will escape next `\` to not escape `'`